projects/student-projects/build-1.md

```diff
@@ -5,9 +5,9 @@
 | Time    | Activity                     |
 | ------- | ---------------------------- |
 | 09:45   | Check-in                     |
-| 10:10 | Feedback Survey |
-| 10:30   | Reach out to Project Mentors |
-| 11:00   | Build              |
+| 10:00   | Feedback Survey |
+| 10:10   | Intro to react-router-dom |
+| 10:30   | React-router-dom workshop |
```

Comment:
```suggestion
| 10:10   | Intro to client-side routing |
| 10:30   | React routing workshop |
```
Can we keep it more generic and avoid referring to a specific npm package? Although we'll be using React Router the concepts apply more broadly